packages/babel-parser/src/plugins/typescript/index.js

```diff
@@ -207,6 +210,8 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         accessibility?: N.Accessibility,
       },
       allowedModifiers: TsModifier[],
+      disallowedModifiers?: TsModifier[],
+      errorTemplate?: string,
     ): void {
       for (;;) {
         const startPos = this.state.start;
```

Comment:
We can pass down `allowedModifiers.concat(disallowedModifier)` to `tsParserModifier` so we don't have to duplicate `disallowedModifier` in the `tsParseModifiers` call.

Reply:
Nit: `allowedModifiers.concat(disallowedModifier ?? [])`